Bobjayafam/politico
# #163665426 Implement create party endpoint

#### What does this PR do?
Implement an endpoint that creates a party and ensures data is stored in a database
#### Description of Task to be completed?
- change the logic of the create party method to use a database instead of a data structure
#### How should this be manually tested?
- clone the GitHub repo
- checkout branch ``ft-create-party-db-163665426``
- create database
- run npm run migration to create tables
- start development server with npm run dev
- on POSTMAN, send a POST request to ``/api/v1/parties``. with form data selected, send the request body to contain name, hqAddress, acronym and logoUrl. logoUrl should be of file type. The response should be the created party with a status code of 201
#### Any background context you want to provide?
The endpoint was only storing data in memory
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/163665426
#### Screenshots (if appropriate)
